ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/ui/graphics/RDXPerceptionVisualizerPanel.java

```diff
@@ -4,46 +4,34 @@
 import com.badlogic.gdx.utils.Array;
 import com.badlogic.gdx.utils.Pool;
 import imgui.ImGui;
-import std_msgs.msg.dds.Empty;
-import us.ihmc.communication.ROS2Tools;
-import us.ihmc.pubsub.DomainFactory;
+import imgui.type.ImString;
 import us.ihmc.rdx.imgui.RDXPanel;
 import us.ihmc.rdx.sceneManager.RDXRenderableProvider;
 import us.ihmc.rdx.sceneManager.RDXSceneLevel;
-import us.ihmc.ros2.ROS2Node;
-import us.ihmc.ros2.ROS2Topic;
 
-import javax.annotation.Nullable;
-import java.util.LinkedHashMap;
+import java.util.Comparator;
+import java.util.Locale;
 import java.util.Set;
+import java.util.TreeSet;
 
 public class RDXPerceptionVisualizerPanel extends RDXPanel implements RDXRenderableProvider
```

Comment:
This guy should be named `RDXPerceptionVisualizersPanel` lol

Reply:
Done